src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.headerField {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  font-family: "Roboto", Arial, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+}
+
+.navList {
+  padding: 0;
+  display: flex;
+  list-style-type: none;
+}
+
+.navItems {
+  display: flex;
+  height: 60px;
```

Comment:
do you need it?

Reply:
No. If I remove these positions, nothing changes🙂

Reply:
remove and try to refactor

Reply:
Something like this?

.navItems {
  display: flex;
  padding: 0;
  margin-right: 20px;
}